preview-src/timeline.tsx

```diff
@@ -76,11 +89,16 @@ const ReviewEventView = (event: ReviewEvent) => {
 					{ event.state === 'PENDING'
 						? <em>review pending</em>
 						: <>
-								reviewed{nbsp}
+								{reviewDescriptor(event.state)}{nbsp}
 								<Timestamp href={event.htmlUrl} date={event.submittedAt} />
 						</> }
 				</Spaced>
 			</div>
+			{
+				event.state !== 'PENDING'
```

Comment:
can we also check that body exists? on .com nothing is shown if there's no content